docs/criufeature.md

```diff
@@ -0,0 +1,93 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# CRIU feature
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** CRIU support is a technical preview feature that is intended for container environments. All APIs and options are subject to change. CRIU support requires the version of CRIU that is a package in Semeru UBI container images.
+
+In Eclipse OpenJ9&trade;, Checkpoint Restore in User Space (CRIU) feature is available as an API that can be used to stop the VM at a particular checkpoint, save the state, and then enable the VM to resume from the point where it was stopped. The time between the VM start and checkpoint is referred to as the checkpoint phase. The checkpointed state is written to one or more image files with the memory content available during the checkpoint phase, such as memory pages, methods, file systems, processes, network connections, and security components. You can restore the checkpoint image from these files in as many environments as possible with improved start-up time and hence with better VM performance.
```

Comment:
This sentence suggests that CRIU isn't an API in other packages. The sentence is missing "the". It's passive voice but it should be active. I suggest something like "The CRIU support includes an API that you can use to..." then same comments as for the similar sentence in version0.38.md.

"The time between..." I'd change slightly. "Checkpoint" is a noun, so we shouldn't use it as a verb (so "checkpointed" isn't a real word, for example). It would take more words, but you could say something like "The period of time between when the VM starts and when you take a checkpoint is..." (Or maybe it should be when the application takes a checkpoint.) Also, "checkpoint phase" should be in italics, since it's a new term. See https://www.eclipse.org/openj9/docs/gc_overview/ for example.

"with" is a bit ambiguous in the next sentence. Do we mean that the state that is written to the image files includes the contents of the memory that was available during the checkpoint phase? (I'm assuming that "was available" is technically different from "existed"). If so, this sentence would be better split into two with that clarification made. I'm a bit unsure about some of the items in the list too. Do we really mean that security components are included in the image? Surely they are anyway, since they're part of the SDK. Or do we mean that memory/data/state/something else that is associated with security components is included?

Same comment as version0.38.md re "as many environments as possible".

Reply:
@tajila - Is the following sentence correct?

"The VM writes the saved state to one or more image files and includes the memory content that existed during the checkpoint phase, such as memory pages, methods, file systems, processes, network connections, and security components."

If we say "includes the memory content" and if it is correct then what else is saved and written to the files?

Is the list of items specified in this sentence correct? If not, then what can we specify as examples?

Reply:
Side note: usages of "VM" should really be "JVM" here

Reply:
> @tajila - Is the following sentence correct?
"The VM writes the saved state to one or more image files and includes the memory content that existed during the checkpoint phase, such as memory pages, methods, file systems, processes, network connections, and security components."

Sounds correct, but I can see how that may be ambiguous. Perhaps

"The checkpoint state is the state of the JVM at the time the checkpoint was initiated. This state includes memory pages, methods, file systems, processes, network connections, and security components. The checkpoint state is written to one or more image files which can be used to restore the JVM. The JVM can be restored many times from the same checkpoint state, and it can be restored in many environments. Start-up benefits are realized by restore a checkpoint image instead of running the JVM from start."

Reply:
@tajila - Thanks! Could you also clarify the doubt that Esther has raised 

"Do we really mean that security components are included in the image? Surely they are anyway, since they're part of the SDK. Or do we mean that memory/data/state/something else that is associated with security components is included?"

Reply:
Security components really refers to security services.